samwise.js

```diff
@@ -5,24 +5,22 @@
  * Released under the MIT license.
```

Comment:
SCRIPT-This block of code could benefit from some comments explaining the logic behind it.